staff_features/resources/step_definitions/resource_shared.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+When 'the user filters by text with the Resource title' do
```

Comment:
it doesn't look like this is called at least not in this PR - there's a separate "When 'the user filters by text with the Resource B title in the modal' do" in resource_merge.rb which looks very similar

Reply:
Yes it not required. It is going to be needed eventually, but it is removed for now.